### PR TITLE
Refactor EVP-LRQA-83367 | Master

### PR DESCRIPTION
**Pull request by MGabriellaS**
[LRQA-83367](https://liferay.atlassian.net/browse/LRQA-83367)

### CODE REVIEW

**Comment by MGabriellaS** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationListing.testcase [hunk 1]
```diff
@@ -42,7 +42,7 @@ definition {
 	@description = "LPS-164728 - Assert that the listing displays the following columns: Organization Name, Status, Country, Contact Name, Contact Phone, Contact Email."
 	@priority = 4
 	test ManagerCanViewColumnsOnListing {
-		task ("Given a manager user with a manager role") {
+		task ("Given a manager user with a manager role is logged") {
```
Comment (heiditrindade) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationListing.testcase:45]: Maybe change to"Given a manager user with a manager role is logged in"

Comment (vinicius-paes) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationListing.testcase:45]: "Please change to Given a user with a manager role is logged"

Comment (vinicius-paes) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationListing.testcase:45]: lets try to keep a pattern when calling the _persona_.

**Review by vinicius-paes — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationListing.testcase (hunk 1 above)
Comment (MGabriellaS) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationListing.testcase]: Ok, thanks for the review, I updated the PR just now

**Comment by MGabriellaS** [comment]
ci:test:sf

**Review by vinicius-paes — APPROVED** [review]

**Comment by vinicius-paes** [comment]
ci:foward

**Comment by vinicius-paes** [comment]
ci:forward

**Comment by MGabriellaS** [comment]
ci:forward

**Pull request closed without merging**